src/test/groovy/org/gradle/profiler/ProfilerIntegrationTest.groovy

```diff
@@ -1305,7 +1305,7 @@ buildGoal {
         logFile.containsOne("* Maven targets: [-v]")
 
         def lines = resultFile.lines
-        lines.size() == 27 // 4 headers, 16 executions, 7 stats
+        lines.size() == 28 // 4 headers, 16 executions, 8 stats
```

Comment:
These tests look like way too much effort to maintain. If we want to keep them I think we should use the sum of named variables instead of comments.

Reply:
I extracted a method which gives the total number of lines depending on the number of executions.